src/components/guide/Guide.jsx

```diff
@@ -0,0 +1,149 @@
+
+const Guide = () => {
+  const questions = [
+    {
+      questionText: "What category is giving you the vibe for tonight?",
+      answerOptions: [
+        { answerText: "Action", id: 28 },
+        { answerText: "Romance", id: 10749 },
+        { answerText: "Science Fiction", id: 878 },
+        { answerText: "Horror", id: 27 },
+      ],
+    },
+    {
+      questionText: "Which do you prefer?",
+      answerOptions: [
+        { answerText: "The Latest Movies", id: 1 },
+        { answerText: "Older Movies", id: 2 },
+      ],
+    },
+    {
+      questionText: "What's the minimum rating you'd prefer?",
+      answerOptions: [
+        { answerText: 8, id: 8 },
+        { answerText: 6, id: 6 },
+        { answerText: 4, id: 4 },
+      ],
+    },
+  ];
+  const apiKey = import.meta.env.VITE_API_KEY;
+  const urlTrending = "https://api.themoviedb.org/3/trending/all/day?api_key=";
+  const [trending, setTrending] = useState([]);
+
+  async function getTrending(signal) {
+    const response = await axios
+      .get(urlTrending + apiKey, signal)
+      .then((res) => res.data);
+    const trends = response.results;
+    setTrending(trends);
+  }
+
+  useEffect(() => {
+    const abortCtrl = new AbortController();
+    const opts = { signal: abortCtrl.signal };
+    getTrending(opts);
+    return () => abortCtrl.abort();
+  }, []);
+
+  const [currentQuestion, setCurrentQuestion] = useState(0);
+  const [guideAnswers, setGuideAnswers] = useState([]);
+  const date = new Date();
+  const year = date.getFullYear() - 3;
+  const categoryAnswer = guideAnswers[0];
+  const latestAnswer = guideAnswers[1];
+  const ratingAnswer = guideAnswers[2];
+  const [isFiltered, setIsFiltered] = useState(false);
+
+  const toggleQuestion = (e) => {
+    if (!guideAnswers.includes(e.target.value))
+      setGuideAnswers([...guideAnswers, e.target.value]);
+
+    const nextQuestion = currentQuestion + 1;
+    if (nextQuestion < questions.length) {
+      setCurrentQuestion(nextQuestion);
+    } else {
+      setIsFiltered(true);
+    }
+  };
+
+  useEffect(() => {
+    let filteredMovies = trending;
+    if (categoryAnswer) {
+      filteredMovies = trending.filter((movie) => {
+        return movie.genre_ids.includes(parseInt(categoryAnswer));
+      });
+    }
+    if (latestAnswer && latestAnswer == 1) {
+      filteredMovies = trending.filter((movie) => {
+        return new Date(movie.release_date) > new Date(year);
+      });
+    } else if (latestAnswer && latestAnswer == 2) {
+      filteredMovies = trending.filter((movie) => {
+        return new Date(movie.release_date) < new Date(year);
+      });
+    }
+    if (ratingAnswer) {
+      filteredMovies = trending.filter((movie) => {
+        return movie.vote_average > ratingAnswer;
+      });
+    }
+    setTrending(filteredMovies);
+  }, [guideAnswers]);
+
+  return (
+    <div className="app">
+      {isFiltered ? (
+        <div className="movie-List mt-5">
```

Comment:
pourquoi une maj ?

Reply:
because isFiltered is a questionable conditional variable
isFiltered as is 
Is the variable filtered or not and since Is and Filtered are 2 different words put together

Reply:
Je parle de `movie-List`